app/views/devise/sessions/new.html.erb

```diff
@@ -30,6 +30,7 @@
           name: 'user[remember_me]',
           id: 'user_remember_me',
           label: t('devise.remember_me'),
+          labelClass: 'clr-white',
```

Comment:
Is this necessary? Why not create a class in the React CSS?

Reply:
It's not necessary, but I just added this flexibility so that we could style the 'Checkbox' component more efficiently according to the page design without editing the 'InputCheckbox' React component. Since we are using this component on multiple pages like add medications page.
![image](https://user-images.githubusercontent.com/15247892/206650587-c2bf2aad-2087-406b-be50-ad38f7438601.png)

Whereas on the 'Sign in' page we need a custom [white] color for the checkbox label.